test/collect_tests.jl

```diff
@@ -94,25 +102,37 @@ end
     daily_agent_aggregate = init_agent_dataframe(model, agent_agg)
     yearly_agent_data = init_agent_dataframe(model, agent_props)
 
-    for year in 1:5
-        for day in 1:365
+    for year = 1:5
```

Comment:
why change `in` to `=`?

Reply:
if you don't like `in` use the `\in<TAB>`, but not equals. Equals should be reserved for assignement.

Reply:
That was the formatter, I've changed the defaults and reverted this already. Will push it soon.